### PR TITLE
Add checked property on items in itemlist

### PR DESCRIPTION
**Pull request by peseb**

### CODE REVIEW

File: turplanlegger/models/list_items.py
Comment (sixcare) [inline turplanlegger/models/list_items.py:1]: Missing from docstring: https://github.com/Turplanlegger/turplanlegger-api/blob/07492feff3637768d072f2e276f7acdad8084273/turplanlegger/models/list_items.py#L12

Missing from __repr__: https://github.com/Turplanlegger/turplanlegger-api/blob/07492feff3637768d072f2e276f7acdad8084273/turplanlegger/models/list_items.py#L51

Missing from mini_serialize: https://github.com/Turplanlegger/turplanlegger-api/blob/07492feff3637768d072f2e276f7acdad8084273/turplanlegger/models/list_items.py#L113

**Review by sixcare — CHANGES_REQUESTED** [review]
Missing quite a lot :(

File: turplanlegger/models/list_items.py
Comment (peseb) [reply turplanlegger/models/list_items.py:1]: Added to mini_serialize, was already in docstring & repr

**Review by sixcare — APPROVED** [review]
GJ 😁

**Pull request merged**